_posts/2021-07-28-Research-Expo.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+header: The Digital Innovation Group @ ASU
+title: Research Expo Workshop
+subtitle:
+images:
+images_caption:
+permalink: /researchexpo/
+author: jdamerow
+excerpt: The Research Computing Expo will be held August 2-6, 2021 by Research Computing. The Digital Innovation Group will be contributing to the expo by hosting two workshops led by Taylor Quinn and Julia Damerow:
```

Comment:
I get the following error:
"YAML Exception reading /Users/jdamerow/diging/_posts/2021-07-28-Research-Expo.md: (<unknown>): mapping values are not allowed in this context at line 10 column 215"
I think it's probably the colon at the end of the sentence. Also, putting the text in quotation marks is probably wise.